core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -135,26 +135,35 @@ contract FeePayer is Testable {
         return StoreInterface(finder.getImplementationAddress(storeInterface));
     }
 
-    // The following methods are used by derived classes to interact with collateral that is adjusted by fees.
-    function _getCollateral(FixedPoint.Unsigned storage rawCollateral)
+    // Gets a user-readable collateral value from a raw value that takes fees into account.
+    function _getCollateral(FixedPoint.Unsigned memory rawCollateral)
         internal
         view
         returns (FixedPoint.Unsigned memory collateral)
     {
         return rawCollateral.mul(cumulativeFeeMultiplier);
     }
 
+    // Converts a user-readable collateral value into a raw value that takes fees into accounts.
```

Comment:
Try clarifying to: "..that accounts for already-assessed fees. If any fees have been taken from this contract, then the raw value will be larger than the user-readable value. "

Reply:
Done.